Index: src/com/company/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.company;\r\n\r\npublic class Main {\r\n\r\n    public static void main(String[] args) {\r\n\t    int a = 100;\r\n        int b=0;\r\n\t    int c = a+b;\r\n        int b = 100;\r\n\t    int c = a + b;\r\n\t    // фвапфвапфвапфвапфвапфвап\r\n        //hi\r\n\r\n    }\r\n}\r\n//ура\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/com/company/Main.java	(revision df0a49fe8ce3c438011d56deeb4d2fbd37e1b4c2)
+++ src/com/company/Main.java	(date 1602089361792)
@@ -7,10 +7,7 @@
         int b=0;
 	    int c = a+b;
         int b = 100;
-	    int c = a + b;
-	    // фвапфвапфвапфвапфвапфвап
-        //hi
-
+	    //не пиши сюда ле
     }
 }
 //ура
